cyhy/mailer/cli.py

```diff
@@ -160,7 +160,7 @@ def get_all_descendants(db, parent):
                 descendants.append(child)
                 descendants += get_all_descendants(db, child)
 
-    return set(descendants)
+    return descendants
```

Comment:
Why not do `list(set(descendants))` here?  That will remove duplicate org ids and still leave us with a list that can be safely passed into the mongo query.

Reply:
Great idea!  Change made in fe19393.